blueocean-commons/src/main/java/io/jenkins/blueocean/commons/BlueOceanConfigProperties.java

```diff
@@ -6,8 +6,10 @@
  * @author Ivan Meredith
  */
 public class BlueOceanConfigProperties {
-    public static final boolean ROLLBAR_ENABLED = Boolean.getBoolean("BLUEOCEAN_ROLLBAR_ENABLED");
+    public static final String BLUEOCEAN_FEATURE_JWT_AUTHENTICATION_PROPERTY = "BLUEOCEAN_FEATURE_JWT_AUTHENTICATION";
+    public static final String BLUEOCEAN_ROLLBAR_ENABLED_PROPERTY = "BLUEOCEAN_ROLLBAR_ENABLED";
```

Comment:
Related somehow??

Reply:
Not related, this had to do with fixing test flakiness.